src/views/datasets/buoy-telemetry/summary.vue

```diff
@@ -181,6 +183,9 @@ const endDate = store.maxDateRaw;
 const startDate = new Date(endDate.valueOf() - 5 * 24 * 60 * 60 * 1000);
```

Comment:
What's the 5 * 24 *60 *1000 for?

Reply:
This math is to set the Date Slider to display for 5 days, and the 24 * 60 * 60 * 1000 is to convert from milliseconds